blockchain/vm/runtime/runtime.go

```diff
@@ -107,7 +107,7 @@ func Execute(code, input []byte, cfg *Config) ([]byte, *state.StateDB, error) {
 		vmenv   = NewEnv(cfg)
 		sender  = vm.AccountRef(cfg.Origin)
 	)
-	cfg.State.CreateSmartContractAccount(address, params.CodeFormatEVM)
+	cfg.State.CreateSmartContractAccount(address, true)
```

Comment:
`CodeFormatEVMConstantinople` was assigned in the previous code, but your change assigns `CodeFormatEVMIstanbul`. Why did you change it?

Reply:
I reverted this code.

Reply:
@aidan-kwon I changed the code that i thought CodeFormat field is deprecated. I reverted this code.